extmod/modtls_mbedtls.c

```diff
@@ -767,6 +771,76 @@ static MP_DEFINE_CONST_OBJ_TYPE(
 /******************************************************************************/
 // ssl module.
 
+#if MICROPY_PY_SSL_ECDSA_SIGN_ALT
+static mp_obj_t ecdsa_sign_alt_cb = mp_const_none;
```

Comment:
```suggestion
static mp_obj_t ecdsa_sign_alt_cb = mp_const_none;
```
This doesn't need to be a root pointer, right ?

Reply:
Yes it will need to be a root pointer.  Eg:
```py
tls.ecdsa_sign_alt(lambda k, b: ...)
```
In that case there is no other reference to the lambda object other than this pointer here, so it must be a root pointer.

Reply:
Good point! It's unlikely to be passed as a lambda, but I made it a root pointer anyway.